File: 2-semestre/uri-online-judge/c/1060.c
#include <stdio.h>

int main() {
    double val1, val2, val3, val4, val5, val6, totPositivos;
    
    scanf("%lf", &val1);
    scanf("%lf", &val2);
    scanf("%lf", &val3);
    scanf("%lf", &val4);
    scanf("%lf", &val5);
    scanf("%lf", &val6);

    totPositivos = 0;

    if (val1 > 0) totPositivos++;
    if (val2 > 0) totPositivos++;
    if (val3 > 0) totPositivos++;
    if (val4 > 0) totPositivos++;
    if (val5 > 0) totPositivos++;
    if (val6 > 0) totPositivos++;

    printf("%.0f valores positivos\n", totPositivos);
    
    return 0;
}

File: 2-semestre/uri-online-judge/java/1060.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
        double[] valores;
        int totPositivos;
        
        Scanner entrada = new Scanner(System.in);
        
        valores = new double[6];
        totPositivos = 0;

        for (int i = 0; i < 6; i++) {
            valores[i] = entrada.nextDouble();

            if (valores[i] > 0) totPositivos++;
        }
        
        System.out.printf("%d valores positivos\n", totPositivos);
    }
}
